File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0208Q01GrdDrg0120Info.java
package nta.med.data.model.ihis.ocsa;

public class OCS0208Q01GrdDrg0120Info {
	private String bogyongCode ;
	private String bogyongName ;
	private String blockGubun ;
	private String bogyongGubun ;
	public OCS0208Q01GrdDrg0120Info(String bogyongCode, String bogyongName,
			String blockGubun, String bogyongGubun) {
		super();
		this.bogyongCode = bogyongCode;
		this.bogyongName = bogyongName;
		this.blockGubun = blockGubun;
		this.bogyongGubun = bogyongGubun;
	}
	public String getBogyongCode() {
		return bogyongCode;
	}
	public void setBogyongCode(String bogyongCode) {
		this.bogyongCode = bogyongCode;
	}
	public String getBogyongName() {
		return bogyongName;
	}
	public void setBogyongName(String bogyongName) {
		this.bogyongName = bogyongName;
	}
	public String getBlockGubun() {
		return blockGubun;
	}
	public void setBlockGubun(String blockGubun) {
		this.blockGubun = blockGubun;
	}
	public String getBogyongGubun() {
		return bogyongGubun;
	}
	public void setBogyongGubun(String bogyongGubun) {
		this.bogyongGubun = bogyongGubun;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0208Q01GrdDrg0120Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCS0208Q01GrdDrg0120Info
	{
		private String _bogyongCode;
		private String _bogyongName;
		private String _blockGubun;
		private String _bogyongGubun;

		public String BogyongCode
		{
			get { return this._bogyongCode; }
			set { this._bogyongCode = value; }
		}

		public String BogyongName
		{
			get { return this._bogyongName; }
			set { this._bogyongName = value; }
		}

		public String BlockGubun
		{
			get { return this._blockGubun; }
			set { this._blockGubun = value; }
		}

		public String BogyongGubun
		{
			get { return this._bogyongGubun; }
			set { this._bogyongGubun = value; }
		}

		public OCS0208Q01GrdDrg0120Info() { }

		public OCS0208Q01GrdDrg0120Info(String bogyongCode, String bogyongName, String blockGubun, String bogyongGubun)
		{
			this._bogyongCode = bogyongCode;
			this._bogyongName = bogyongName;
			this._blockGubun = blockGubun;
			this._bogyongGubun = bogyongGubun;
		}

	}
}